src/lib/db/query.js

```diff
@@ -2,42 +2,36 @@
 
 const db = require('./postgres');
 
-const countsByType = 'SELECT dataset, count(*)::INTEGER FROM lev_audit WHERE date_time > $(from)';
-const countsByUser =
-  'SELECT date_time::DATE AS date, dataset, username, count(*)::INTEGER FROM lev_audit WHERE date_time > $(from)';
-const until = 'AND date_time < $(to)';
-const groupByType = ' GROUP BY dataset';
-const groupByDateTypeUser = ' GROUP BY date_time::date, dataset, username ORDER BY date_time::date';
 const groupByTypeGroup = 'GROUP BY name, dataset';
 const totalCount = 'SELECT count(*)::INTEGER FROM lev_audit';
-const forToday = ' WHERE date_time::DATE = current_date';
-const fromDate = 'date_time::DATE >= $(from)';
-const toDate = 'date_time::DATE < $(to)';
+const forToday = ' WHERE date_time >= (current_date::date)::timestamp';
```

Comment:
It feels like we should defend ourselves against dates in the future here but I suppose that can be sorted out later.

Reply:
I'm not sure I understand. This is a "_right side only_" date, so it shouldn't slow anything down. Actually now I think about it, this should be handled the same way as the rest of the date/time stuff, i.e. with a proper timestamp... I'll update...

Reply:
fixed

Reply:
I meant that `forToday` should test that the time is between the start of today and the end of today. i.e. It needs two clauses and not just one.

Of course this works because we don't expect data from the future to be present.

Reply:
yes, unless @aarongill1 has been time-travelling again!

Reply:
Exactly.